src/components/Mandal/MandalHeader/GoBackButton.js

```diff
@@ -38,7 +38,7 @@ export default function GoBackButton() {
   return (
     <GoBackContainer onClick={handleGoBackButtonClick}>
       <div className="tooltip">
-        <img src="/icons/back.svg" />
+        <img src="/icons/back.svg" alt="go-back-button" />
         <span className="tooltiptext ">뒤로가기</span>
```

Comment:
수진님께서 작업하신 내용은 아니지만 마침 눈에 띄는 불필요한 공백이 있어서, 수정 및 커밋 부탁드릴 수 있을까요? 🙇‍♂️

```suggestion
        <span className="tooltiptext">뒤로가기</span>
```

Reply:
넵 수정해서 커밋하도록 하겠습니다🙇‍♀️